schema/release-schema.json

```
@@ -1033,11 +1033,13 @@
         },
         "format": {
           "title": "Format",
-          "description": "The format of the document, using the open [IANA Media Types](https://www.iana.org/assignments/media-types/media-types.xhtml) codelist (see the values in the 'Template' column), or using the 'offline/print' code if the described document is published offline. For example, web pages have a format of 'text/html'.",
+          "description": "The format of the document, using the open [media type](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#media-type) codelist. For example, web pages have a format of 'text/html'. Use the 'offline/print' code if the document is published offline.",
```

Avoid imperative sentences that are instructions to publishers:

```suggestion
          "description": "The format of the document, using the open [media type](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#media-type) codelist. For example, a web page has a format of 'text/html'. The 'offline/print' code is reserved for printed documents not available in other formats.",
```